packages/sol/tests/verification.test.ts

```diff
@@ -0,0 +1,137 @@
+    ).toBe(FieldError.TutorBioTooLong);
+  });
+});
+
+describe("tutor about validation", () => {
+  it("should accept about in range 1-1000 chars", () => {
+    expect(
+      isValidTutorAbout(
+        "<div><h3>about john doe</h3><p>A tutor guides students, helps with learning, explains concepts simply, and improves understanding step by step.</p></div>"
+      )
+    ).toBe(true);
+  });
+  it("should reject empty tutor about", () => {
+    expect(isValidTutorAbout("")).toBe(FieldError.TutorAboutEmpty);
+  });
+  it("should reject tutor about with more than 1000 letters", () => {
+    expect(
+      isValidTutorAbout(
+        "A tutor plays a vital role in enhancing a student's learning experience by providing personalized guidance and support. Unlike a traditional classroom setting, where the teacher's attention is divided among many students, a tutor can focus on the individual needs of each learner. This allows for a customized approach that aligns with the student's learning style, pace, and specific challenges. Whether a student is struggling with a particular subject or simply seeking to improve their skills, a tutor adapts lessons to suit their abilities, offering step-by-step explanations and targeted practice.One of the most important aspects of tutoring is its ability to build confidence. Many students feel anxious or overwhelmed in large classroom environments, especially if they fall behind. A tutor creates a safe, non-judgmental space where students can ask questions freely and work through problems at their own speed. This personalized attention helps clarify difficult concepts and ensures that the student fully grasps the material before moving on to more complex topics.In addition to academic instruction, tutors help students develop essential skills such as time management, study techniques, and problem-solving. These skills are not only valuable for immediate academic success but also for future educational and professional pursuits. A tutor teaches students how to approach their work systematically, breaking down larger tasks into manageable steps, which leads to better productivity and less stress.Moreover, tutoring is beneficial for students of all ages. For younger learners, a tutor helps lay a strong foundation in critical subjects like reading and math. For older students, tutors offer specialized support in more advanced topics, such as science, mathematics, or languages. Tutoring can also be crucial for exam preparation, where tutors guide students through practice tests, review key concepts, and teach test-taking strategies, helping them feel more prepared and confident.In conclusion, tutors play a key role in enhancing a student's academic journey. They provide personalized, focused instruction that helps students overcome challenges, build confidence, and achieve their educational goals. Through consistent support, a tutor not only improves a student's grades but also fosters a lifelong love for learning."
```

Comment:
Don't use this entire test block. Use a package like `fakerjs` to generate fake text for you.